docs/sections/getting_started.md

```diff
@@ -170,6 +176,18 @@ Trigger an execution by pushing to github:
 git push
 ```
 
+## More Information
```

Comment:
We can remove this section and add it to the note above in `Run your Workflow`. The `More Information` title might be confused to where to get more information about what was covered in the Getting started guide (as opposed to more information about the CLI).

Reply:
I'd redo this too thanks.